src/configure/configure.ts

```diff
@@ -451,9 +488,32 @@ class PipelineConfigurer {
                 .then((webApps) => webApps.map(x => { return { label: x.name, data: x }; })),
             { placeHolder: Messages.selectWebApp },
             TelemetryKeys.WebAppListCount);
+
+        await this.validateIfPipelineCanBeSetupOnResource(selectedResource.data.id);
         this.inputs.targetResource.resource = selectedResource.data;
     }
 
+    private async updateScmType(queuedPipeline: Build) {
+        await this.appServiceClient.updateScmType(this.inputs.targetResource.resource.id);
+
+        let metadata = await this.appServiceClient.getSiteMetadata(this.inputs.targetResource.resource.id);
+        let organizationId = await this.azureDevOpsClient.getOrganizationIdFromName(this.inputs.organizationName);
+        metadata["properties"] = {
```

Comment:
you are only adding additional properties, so I assume this is a patch call and not post

Reply:
Metadata doesn't seem to support patch call. We however preserve other properties of metadata, as we change only the ones we require.